python/hail/representation/interval.py

```diff
@@ -122,6 +124,7 @@ def contains(self, locus):
         return self._jrep.contains(locus._jrep)
 
     @handle_py4j
+    @typecheck_method(interval=Interval)
```

Comment:
does this work? I thought this is a python name error. If it crashes the tests, change this to `anytype` for now.